### DISCUSSION TITLE
Can't duplicate the example

### DISCUSSION THREAD

**Discussion by macdroid53** [j8sr0230/fc_nodes#110] [Q&A]
As seen below it appears the Eval surf is returning the point correctly. This implies the Shape Content is working (I note this node changed names recently. This is not reflected in the github readme.) But, the normal apparently is not output correctly, so the box is in space.

I attempted to look at the code for Eval surf, but, with no comments, I couldn't make heads or tails of what might be happening.

![align_shape-2](https://user-images.githubusercontent.com/6831752/205498216-698b735b-7f02-4a9a-9ab8-b4b78845c7ef.gif)

**Comment by j8sr0230** [comment]
Hello, I have reworked the Align node again and made it a bit more user friendly. The Align node now has a pivot socket that allows you to easily define the pivot point of the rotation. Alternatively, you can use the new Dir(ection) socket of the Box node, with which you could directly align the generated Box instances to the sphere surface (see attachment).
![align_boxes_node_graph](https://user-images.githubusercontent.com/39261150/205989504-66e548d9-e222-4d19-90ef-3ed37fd1e0aa.PNG)
![align_boxes_node_graph_result](https://user-images.githubusercontent.com/39261150/205989519-a879cb37-eb2c-4c51-93bf-6bbe036b8a95.PNG)

**Reply by macdroid53** [reply]
I'd try to rework and give it a go...but, as you'll note in my other posts, as of the update from Addon manager I can't load any previously created json file.
